packages/core/demo/data/toolbar.ts

```diff
@@ -0,0 +1,58 @@
+import * as barChart from "./bar";
+import * as lineChart from "./line";
+
+// utility function to enable toolbar option
+const addToolbarOptions = (options, configs?) => {
+	options.experimental = true;
+
+	options.toolbar = {
+		enabled: true
+	};
+	options.zoomBar = {
+		top: {
+			enabled: true
+		}
+	};
+	if (configs) {
+		if (configs.titlePostifx) {
+			options.title += configs.titlePostifx;
+		}
+		if (configs.maxIcons) {
+			options.toolbar.maxIcons = configs.maxIcons;
```

Comment:
```suggestion
			options.toolbar.numberOfIcons = configs.numberOfIcons;
```

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/791/commits/f6a4a5976a3106e01e7907c24c082647153b9858.